mlflow/sagemaker/__init__.py

```diff
@@ -32,7 +33,6 @@
 from mlflow.models.model import MLMODEL_FILE_NAME
 from mlflow.protos.databricks_pb2 import INVALID_PARAMETER_VALUE, RESOURCE_DOES_NOT_EXIST
 from mlflow.tracking.artifact_utils import _download_artifact_from_uri
-from mlflow.utils import get_unique_resource_id
```

Comment:
It seems `get_unique_resource_id` is only used for this ID generation, can we remove it?

Reply:
thanks for the catch, it's never used now.